src/components/DocumentInfo/AddressSingle/AddressSingle.module.scss

```diff
@@ -0,0 +1,40 @@
+@import "../../../styles/abstracts/__abstracts.scss";
```

Comment:
Could you point me to the documentation about storybook and which addon handles the module files correctly ?

Reply:
https://storybook.js.org/docs/configurations/custom-webpack-config/

Reply:
for me
- remove module extension
- check @storybook/preset-create-react-app